src/components/SecondaryPanes/index.js

```diff
@@ -107,7 +179,105 @@ class SecondaryPanes extends Component<Props> {
         : L10N.getStr("breakpoints.disable")
     };
 
-    return <input {...inputProps} />;
+    if (!features.breakpointsDropdown) {
+      return <input {...inputProps} />;
+    }
+
+    return (
+      <div className="breakpoints-buttons">
+        {this.renderBreakpointsDropdown()}
+        <input {...inputProps} />
+      </div>
+    );
+  }
+
+  renderPauseButton() {
+    const { pauseData, breakOnNext, isWaitingOnBreak } = this.props;
+
+    if (pauseData) {
+      return debugBtn(
+        this.props.resume,
+        "resume",
+        "active",
+        L10N.getFormatStr("resumeButtonTooltip", formatKey("resume"))
+      );
+    }
+
+    if (isWaitingOnBreak) {
+      return debugBtn(
+        null,
+        "pause",
+        "disabled",
+        L10N.getStr("pausePendingButtonTooltip", true)
+      );
+    }
+
+    return debugBtn(
+      breakOnNext,
+      "pause",
+      "active",
+      L10N.getFormatStr("pauseButtonTooltip", formatKey("pause"))
+    );
+  }
+
+  /*
+   * The pause on exception button has three states in this order:
+   *  1. don't pause on exceptions      [false, false]
+   *  2. pause on uncaught exceptions   [true, true]
+   *  3. pause on all exceptions        [true, false]
+  */
+  renderPauseOnExceptions() {
+    return debugBtn(
+      () => actions.pauseOnExceptions(false, false),
+      "pause-exceptions",
+      "all enabled",
+      L10N.getStr("pauseOnExceptions")
+    );
+  }
+
+  renderPauseOnUncaughtExceptions() {
+    return debugBtn(
+      () => actions.pauseOnExceptions(true, false),
+      "pause-exceptions",
+      "uncaught enabled",
+      L10N.getStr("pauseOnUncaughtExceptions")
+    );
+  }
+
+  renderIgnoreExceptions() {
+    return debugBtn(
+      () => actions.pauseOnExceptions(true, true),
+      "pause-exceptions",
+      "enabled",
+      L10N.getStr("ignoreExceptions")
+    );
+  }
+
+  renderBreakpointsDropdown() {
+    const Panel = (
+      <ul>
+        <li> {this.renderPauseButton()} Pause on Next Statement</li>
+        <li>
+          {" "}
+          {this.renderPauseOnUncaughtExceptions()}
+          {L10N.getStr("pauseOnUncaughtExceptions")})
+        </li>
+        <li>
+          {" "}
+          {this.renderPauseOnExceptions()}
+          {L10N.getStr("pauseOnExceptions")}
+        </li>
+        <li>
+          {" "}
+          {this.renderIgnoreExceptions()}
+          {L10N.getStr("ignoreExceptions")}
+        </li>
+      </ul>
+    );
+
+    return (
+      <Dropdown class="dropdown" panel={Panel} icon={<Svg name="plus" />} />
+    );
```

Comment:
Yep... we'll need to move this into it's own component now :) it's too damn big

`SecondaryPanes/BreakdpointsDropdown`

```js
export default function renderBreakpointsDropdown() {
  // stuff
  return <Dropdown class="dropdown" panel={Panel} icon={<Svg name="plus" />} />
}
```